scripts/package.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# package.sh by Andrzej Pogonowski
+# What is this?: This is the script that packages all the necessary files for deployment
+# Why does this exists?: The final deliverable is thicc so we need to limit the amout of files that are passed
```

Comment:
I don't think this is an appropriate description 🤔

Reply:
Yea I need to turn on the spell check